src/styles/code-read.css

```
@@ -39,64 +42,445 @@
+  justify-content: center;
+  margin: calc(12px * var(--pr-scale, 1)) 0
+    calc(8px * var(--pr-scale, 1));
+  pointer-events: none;
+}
+
+.frame53-pr-settings-mascot {
+  display: block;
+  width: min(
+    100%,
+    calc(
+      var(--pr-intro-mascot-width-px, 168) * var(--pr-scale, 1) *
```

Stylelint (17.10.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)